test/client/doc.js

```diff
@@ -474,5 +474,74 @@ describe('Doc', function() {
         }
       ], done);
     });
+
+    describe('with an op collision', function() {
+      function testScenario(preventInvertInRollback, done) {
+        var doc1 = this.backend.connect().get('dogs', 'snoopy');
+        var doc2 = this.backend.connect().get('dogs', 'snoopy');
+        doc2.preventInvertInRollback = preventInvertInRollback;
+
+        var pauseSubmit = false;
+        var fireSubmit;
+        var invertHaveBeenCalled = false;
```

Comment:
I can't help but notice a grammar issue. Suggest to simplify this name.
```suggestion
        var invertCalled = false;
```